src/js/actions/export.js

```diff
@@ -163,11 +166,44 @@ define(function (require, exports) {
                     filePath: "",
                     status: ExportAsset.STATUS.ERROR
                 };
-                return this.transfer(updateExportAsset, document, layer, assetIndex, assetProps);
+                return this.transfer(updateExportAsset, document, _layers, assetIndex, assetProps);
             }) ;
     };
 
     /**
+     * Insert an asset (or assets) into a document or set of layers, by inserting into the existing list
+     * at the given index.
+     * Sync to PS metadata afterwards
+     *
+     * 
```

Comment:
@shaoshing is gonna smack you if you try to check in two empty lines in a row here.